doc8/checks.py

```diff
@@ -60,12 +60,16 @@ def report_iter(self, line):
                 yield ("D003", "Tabulation used for indentation")
 
 
-class CheckCarriageReturn(LineCheck):
+class CheckCarriageReturn(ContentCheck):
     REPORTS = frozenset(["D004"])
 
-    def report_iter(self, line):
-        if "\r" in line:
-            yield ("D004", "Found literal carriage return")
+    def __init__(self, cfg):
+        super(CheckCarriageReturn, self).__init__(cfg)
```

Comment:
This ``__init__`` is unnecessary given that it's simply calling the superclass implementation, right?

Reply:
Possibly. I just followed the pattern of `CheckNewlineEndOfFile` which has the same `__init__`. I'll check this.

Reply:
I'd removed the `__init__` as suggested.